aws_lambda_builders/actions.py

```diff
@@ -92,10 +92,16 @@ class CopySourceAction(BaseAction):
 
     PURPOSE = Purpose.COPY_SOURCE
 
-    def __init__(self, source_dir, dest_dir, excludes=None):
+    def __init__(self, source_dir, dest_dir, only=None, excludes=None):
         self.source_dir = source_dir
         self.dest_dir = dest_dir
+        self.only = only
```

Comment:
Can you describe what you the use-case for this is vs just using the excludes? Should we allow only and excludes to be passed in? If so, how do they play with each other?